src/NzbDrone.Api/NzbDrone.Api.csproj

```diff
@@ -194,11 +198,11 @@
     <Compile Include="NzbDroneRestModule.cs" />
     <Compile Include="NzbDroneRestModuleWithSignalR.cs" />
     <Compile Include="PagingResource.cs" />
-    <Compile Include="Profiles\Languages\LanguageModule.cs" />
-    <Compile Include="Profiles\Languages\LanguageResource.cs" />
+    <Compile Include="LanguageProfiles\Languages\LanguageModule.cs" />
+    <Compile Include="LanguageProfiles\Languages\LanguageResource.cs" />
     <Compile Include="Profiles\LegacyProfileModule.cs" />
-    <Compile Include="Profiles\ProfileModule.cs" />
-    <Compile Include="Profiles\ProfileResource.cs" />
+    <Compile Include="LanguageProfiles\languageProfileModule.cs" />
```

Comment:
I think `Profiles\Quality` and `Profiles\Language` makes sense to locate these two somewhat related items.